src/datepicker/datepicker-navigation-select.ts

```diff
@@ -18,13 +19,22 @@ import {NgbCalendar} from './ngb-calendar';
     }
   `],
   template: `
-    <select [disabled]="disabled" class="custom-select d-inline-block" [value]="date?.month" (change)="changeMonth($event.target.value)">
-      <option *ngFor="let m of months" [value]="m">{{ i18n.getMonthShortName(m) }}</option>
-    </select>` +
-      `<select [disabled]="disabled" class="custom-select d-inline-block" [value]="date?.year" (change)="changeYear($event.target.value)">
-      <option *ngFor="let y of years" [value]="y">{{ y }}</option>
+    <select
+      [disabled]="disabled"
+      class="custom-select d-inline-block"
+      [value]="date?.month"
+      (change)="changeMonth($event.target.value)"
+      tabindex="-1">
```

Comment:
In the current implementation its the `ngb-datepicker` itself that captures all events and prevents default behaviour to avoid page scrolling when hitting arrows, space, etc.

Now that I think of, it might be better in future to focus only a specific date instead of everything, so the default navigation behaviour with keyboard select boxes and arrows would continue working as before